sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py

```diff
@@ -28,6 +28,7 @@ def __init__(
             *,
             additionally_allowed_tenants: Optional[List[str]] = None,
             allow_broker: Optional[bool] = None,
+            parent_window_handle: Optional[int]=None,
```

Comment:
small formatting thing:

```suggestion
            parent_window_handle: Optional[int] = None,
```